pkg/report/writer.go

```diff
@@ -116,8 +117,10 @@ func (tw TableWriter) write(result Result) {
 	}
 
 	var results []string
-	for _, severity := range dbTypes.SeverityNames {
-		r := fmt.Sprintf("%s: %d", severity, severityCount[severity])
+
+	for _, severity := range utils.UniqueSeverities(tw.Severities) {
+		severityName := severity.String()
+		r := fmt.Sprintf("%s: %d", severityName, severityCount[severityName])
```

Comment:
I've come up with a better idea.

```suggestion
	var severities []string
	for _, sev := range tw.Severities {
		severities = append(severities, sev.String())
	}

	for _, severity := range dbTypes.SeverityNames {
		if !utils.StringInSlice(severity, severities) {
			continue
		}
		r := fmt.Sprintf("%s: %d", severity, severityCount[severity])
		results = append(results, r)
	}
```

In this way, we don't need to remove duplicates and sort severities.

Reply:
Nice, no need to add new utils